docs/docsource/reference/apis/iam/create_access_key.rst

```diff
@@ -0,0 +1,63 @@
+.. _CreateAccessKey:
+
+CreateAccessKey
```

Comment:
Somehow we have to mention, if a username is not specified with all these calls:
* `CreateAccessKey`
* `UpdateAccessKey`
* `DeleteAccessKey`
* `ListAccessKeys`

If an admin makes the above API calls with account credentials without the parameter `UserName` then the action is on itself.
Similar for the user. If we use user credentials with proper policies to call create access key without a `UserName` parameter, it will create AK/SK for itself.

Reply:
Actually, it's mentioned below in the description of the AWS cli part.

Reply:
That sounds right as discussed over google meet.